src/services/service.ts

```diff
@@ -0,0 +1,167 @@
+        console.log(e)
+        throw (e)
+    }
+
+}
+
+const createTicket = async (request: Express.Request, response: Express.Response) => {
+    const { price, category, match_date, stadium, created } = request.body
+    try {
+        await client.query('INSERT INTO ticket ( price, category, match_date, stadium, created) VALUES ($1, $2, $3, $4, $5) RETURNING *', [ price, category, match_date, stadium, created], (error: Error, results: QueryResult) => {
+            if (error) {
+                throw error
+            }
+            response.status(201).send(`Ticket added with ID ${results.rows[0].ticket_id}, name: ${results.rows[0].price}, category: ${results.rows[0].category}, match_date: ${results.rows[0].match_date}, stadium: ${results.rows[0].stadium}, created: ${results.rows[0].created}`)
+        })
+    } catch (e) {
+        console.log(e)
+        throw (e)
+    }
+
+}
+
+//PUT
+
+const updateUser = async (request: Express.Request, response: Express.Response) => {
+    const id = parseInt(request.params.id)
+    const { name, email, cellphone, age, address, country, state } = request.body
+    try {
+        await client.query('UPDATE users SET name = $1, email = $2, cellphone = $3, age= $4, address = $5, country = $6, state = $7 WHERE user_id = $8', [name, email, cellphone, age, address, country, state, id], (error: Error, results: QueryResult) => {
+            if (error) {
+                throw error
+            }
+            response.status(200).send(`User modified with ID: ${id}, name: ${name}, email: ${email}, cellphone: ${cellphone}, age: ${age}, address: ${address}, country: ${country}, state: ${state}`)
+        }
+        )
+    } catch (e) {
+        console.log(e)
+        throw (e)
+    }
+}
+
+const updateTicket = async (request: Express.Request, response: Express.Response) => {
+    const id = parseInt(request.params.id)
+    const { price, category, match_date, stadium, created } = request.body
+    try {
+        await client.query('UPDATE ticket SET price = $1, category = $2, match_date = $3, stadium= $4, created = $5 WHERE ticket_id = $6', [price, category, match_date, stadium, created, id], (error: Error, results: QueryResult) => {
+            if (error) {
+                throw error
+            }
+            response.status(200).send(`User modified with ID: ${id}, price: ${price}, category: ${category}, match_date: ${match_date}, stadium: ${stadium}, created: ${created}`)
+        }
+        )
+    } catch (e) {
+        console.log(e)
+        throw (e)
+    }
+}
+
+//DELETE
+
+const deleteUser = (request: Express.Request, response: Express.Response) => {
+    const id = parseInt(request.params.id)
+    try {
+        client.query('DELETE FROM users WHERE user_id = $1', [id], (error: Error, results: QueryResult) => {
+            if (error) {
+                throw error
+            }
+            response.status(200).send(`User deleted with ID: ${id}`)
+        })
+    } catch (e) {

```

Comment:
maybe for more legible e can change for error

Reply:
done